ScnSoft-Education/LeetCodeTraining/ReverseInteger7/ReverseIntegerSolution.cs

```diff
@@ -0,0 +1,26 @@
+﻿namespace LeetCodeTraining.ReverseInteger
+{
+    //https://leetcode.com/problems/reverse-integer/description/
+    public class ReverseIntegerSolution
+    {
+        public static int Reverse(int x)
+        {
+            int reversedX = 0;
+            while (x != 0)
+            {
+                int digit = x % 10;
+                x /= 10;
+                if (reversedX > int.MaxValue / 10 || (reversedX == int.MaxValue / 10 && digit > 7))
+                {
+                    return 0;
+                }
+                if (reversedX < int.MinValue / 10 || (reversedX == int.MinValue / 10 && digit < -8))
```

Comment:
Can you please combine this condition with previous one to get rid one more return of 0?